shell/src/main/java/com/sequenceiq/cloudbreak/shell/commands/common/SecurityGroupCommands.java

```diff
@@ -36,44 +36,53 @@ public boolean createAvailable() {
     public String create(
             @CliOption(key = "name", mandatory = true, help = "Name of the security group") String name,
             @CliOption(key = "description", help = "Description of the security group") String description,
-            @CliOption(key = "rules", mandatory = true,
+            @CliOption(key = "existingGpId", help = "Existing security group id.") String existingSecurityGroupId,
```

Comment:
this class is not platform specific. Please check how we implemented like the network commands.